front/src/applications/operationalStudies/components/MacroEditor/NGE.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useRef } from 'react';
+
+/* eslint-disable import/extensions, import/no-unresolved */
+import ngeMain from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/main.js?url';
+import ngePolyfills from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/polyfills.js?url';
+import ngeRuntime from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/runtime.js?url';
+import ngeStyles from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/styles.css?url';
+import ngeVendor from '@osrd-project/netzgrafik-frontend/dist/netzgrafik-frontend/en/vendor.js?url';
+/* eslint-enable import/extensions, import/no-unresolved */
+
+const frameSrc = `
+<!DOCTYPE html>
+<html class="sbb-lean">
```

Comment:
The `sbb-light` CSS class can be added here to fix the broken menus when the browser is configured to prefer dark themes.

See https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/pull/176